### ISSUE TITLE
Commands - ejectUser

### ISSUE THREAD

**Issue by TiagoJacobs** [bigbluebutton/bigbluebutton-html-plugin-sdk#30]
We want to create a new type of plugin hook "commands", that will allow the plugin to perform an action on behalf of the user, like ejecting an user from the meeting (that will be the initial implementation).

Sample implementation:
- create a button on screen (you choose where) and when the button is clicked, you eject all users on the meeting, except the current one - only display this button if the current user is a moderator.

**Comment by GuiLeme** [comment]
Good point, with the current structure of the PluginApi we could create this as a serverCommand, and it would need approve from server-side, perhaps.

**Comment by gustavotrott** [comment]
I have a use case where it would be good a function to "Leave meeting".
Alternatively it can be done via mutation, providing a `useMutation` hook should do the trick.

**Comment by gustavotrott** [comment]
Should it have different categories of hooks?
for exemple 
- `pluginApi.moderatorCommands`
- `pluginApi.presenterCommands`
- `pluginApi.userCommands`
